src/amp/components/elements/VideoVimeoBlockComponent.tsx

```diff
@@ -15,6 +15,7 @@ export const VideoVimeoBlockComponent: React.FC<{
                 layout="responsive"
                 width={element.width}
                 height={element.height}
+                do-not-track="do-not-track"
```

Comment:
I presume sending this attribute even if it's not supported won't have any adverse side effects?

Reply:
It shouldn't, but Im intending to hold off merging this change until the PR for the changes to amp-html are live.